### ISSUE TITLE
Example code is not displayed on Utils > Request page

### ISSUE THREAD

**Issue by BayBreezy** [unjs/h3#877]
### Describe the change

Upon visiting the following page, none of the code examples are being displayed.
URL: https://h3.unjs.io/utils/request
SS:
<img width="1420" alt="image" src="https://github.com/user-attachments/assets/79e25449-c58c-4e19-af02-a9bbe7af124d">

### URLs

https://h3.unjs.io/utils/request

### Additional information

- [ ] Would you be willing to help?

**Comment by BayBreezy** [comment]
Ok. So all the examples are bundled at the bottom of the page 🙈

**Comment by achou11** [comment]
yeah this is affecting variois pages. going to hazard a guess that this is an issue closer to https://github.com/unjs/undocs

maybe similar to https://github.com/unjs/undocs/issues/132?
